src/utils/file-processor.js

```diff
@@ -97,9 +97,8 @@ const fileToChunks = (file, handle, opts = {}) =>
 
       Promise.all(chunks.map(readBlob)).then(arrayBuffer => {
         let encryptedChunks = arrayBuffer
-          .map(arrayBufferToString)
-          .map((binaryString, idx) =>
-            Encryption.encryptChunk(handleInBytes, idx + 1, binaryString)
+          .map((arrayBuffer, idx) =>
+            Encryption.encryptChunk(handleInBytes, idx + 1, arrayBuffer)
```

Comment:
The forge encryption algs can take arrayBuffers or byte strings (and several other input types) with no modification needed to our encryptChunk method.